modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -10,12 +10,18 @@ After you provision the adapter, you must create an API to bind to the service.
 +
 If you want to create REST or SOAP APIs with OAS or WSDL definition, use Anypoint Platform. 
 +
-Note that if you want to upload a REST API or OAS asset type, use a .json or .yaml file. Alternatively, if you want to upload a SOAP API or WSDL asset type, use a .wsdl or .xml file.
-* <<create-api-ASM-CLI, Create an API using Anypoint Service Mesh Autocreate>>
```

Comment:
Since you say "use a .wsdl..." later on, I suggest removing "you must" for consistency.

Reply:
Done! Thanks!